<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UserRoleUI</class>
 <widget class="QWidget" name="UserRoleUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Roles</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="mAddButton" >
         <property name="maximumSize" >
          <size>
           <width>35</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text" >
          <string>-></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mRemoveButton" >
         <property name="maximumSize" >
          <size>
           <width>35</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text" >
          <string>&lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>211</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="2" >
      <widget class="QListWidget" name="mSelectedList" />
     </item>
     <item row="1" column="0" >
      <widget class="QListWidget" name="mAvailableList" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel6" >
       <property name="text" >
        <string>Available Roles:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="textLabel5" >
       <property name="text" >
        <string>Selected Roles:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>171</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>170</y>
     <width>429</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="" />
  <widget class="QWidget" name="" />
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>mAddButton</tabstop>
  <tabstop>mRemoveButton</tabstop>
  <tabstop>mOKButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
